src/test/resources/pages/adminEmailCompose.html

```diff
@@ -27,7 +27,7 @@
          </script>
          <script src="/tinymce/js/tinymce/tinymce.min.js" type="text/javascript">
          </script>
-         <link href="http://localhost:8888/tinymce/js/tinymce/skins/lightgray/skin.min.css" rel="stylesheet"/>
```

Comment:
It only happens once you stage the app on your own server. The app.url variable is defined in the web.xml file and is the server to which you will stage your app. Since many of us do not do the staging, the url in app.url is localhost and hence in your case it has been subsituted by ${app.url} by godmode